content/2_Bronze/Rect_Geo.mdx

```diff
@@ -601,19 +601,23 @@ print(rects[0].area() + rects[1].area() - intersect(rects[0], rects[2]) - inters
 Certain tasks show up often in rectangle geometry problems. For example, many problems
 involve finding the overlapping area of two or more rectangles based on their coordinate points, 
 or determining whether two rectangles intersect. Here, we'll discuss these formulas.
-	
+
 Note that these formulas only apply to rectangles which have sides parallel to the coordinate axes.
 
 <Info title="Coordinates">
-A rectangle can be represented with $2$ points: its top right corner and bottom left corner. We'll label these points $tr$ (top right) and $bl$ (bottom left).
+A rectangle can be represented with $2$ points: its top right corner and bottom left corner.
+We'll label these points $tr$ (top right) and $bl$ (bottom left).
+
+Note that it doesn't really matter where you place the origin where the points are relative
+to, as long as you stay consistent.
```

Comment:
^ this comment isn't necessary imo

Reply:
how 2 address #2593 then?

Reply:
Say something like - In this module, we assume that increasing $x$ moves right and increasing $y$ moves up.